### ISSUE TITLE
Make CSS building super duper fast

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#7957]
We spend a substantial amount of the startup overhead in "waiting for the CSS to be finalized", so it would be a great if we can sit down and figure out how to rewrite our build tasks so that CSS happens:

- [ ] in parallel
- [ ] as early as possible
- [ ] as fast as possible

**Comment by Pomax** [comment]
Let's all tackle this together over zoom, and then it might be good for @tbrlpld to sit in

**Comment by tbrlpld** [comment]
@Pomax sounds good to me 👍

**Comment by tbrlpld** [comment]
Re: parallel

Since we already have `npm-run-all` installed, we can use `run-p` to run in parallel.

https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/run-p.md

**Comment by bheasman** [comment]
Agreed to have a timeboxed meeting together as devs to think this one through (1 hr)

**Comment by tbrlpld** [comment]
Who's seeting up the meeting?

**Comment by tbrlpld** [comment]
Should we be closing this one now? 

I guess the only related work that is open is finishing the Tailwind transition.

**Comment by bheasman** [comment]
Agreed to close this ticket for now - will raise a new ticket once the Tailwind work has been completed if necessary
